### ISSUE TITLE
Bundler workers are not exported from package anymore

### ISSUE THREAD

**Issue by kouak** [visgl/loaders.gl#3034]
I'm using deck.gl with `@loaders.gl/mvt` for an internal app, in an offline/airgapped environment. Browsers are not connected to the internet and can't load the worker bundle from unpkg.com.

I used to import the worker bundle with a custom webpack config and the following statement :
```typescript
// Deck.tsx
import mvtLoaderWorkerUrl from '@loaders.gl/mvt/dist/mvt-worker.js';

// webpack.config.ts
            {
              test: [/dist\/mvt-worker\.js$/],
              type: 'asset/resource',
              generator: {
                filename: 'mvt-worker.[hash].js',
              },
            }
```

For reference, here's a similar implementation with vite (no special config) :
```typescript
import mvtLoaderWorkerUrl from '@loaders.gl/mvt/dist/mvt-worker.js?url';
```

With `@loaders.gl/mvt` v4.2.2, I'm encountering this error :
```
Module not found: Error: Package path ./dist/mvt-worker.js is not exported from package <appDir>/frontend/node_modules/@loaders.gl/mvt (see exports field in <appDir>/frontend/node_modules/@loaders.gl/mvt/package.json)
```

A workaround is to import the file via relative import
```typescript
import mvtLoaderWorkerUrl from '../../../../node_modules/@loaders.gl/mvt/dist/mvt-worker.js';
```

I guess a solution would be to include the bundled worker.js file in the `package.json#exports` field:
```json
{
 "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "import": "./dist/index.js",
      "require": "./dist/index.cjs"
    },
    "./dist/mvt-worker.js": {
      "import": "./dist/mvt-worker.js"
    }
  }
}
```

I'm currently having this issue with `@loaders.gl/mvt` because that's the one I'm using, but AFAIK, this issue should affect all other loaders.

**Comment by ibgreen** [comment]
Yes this seems like a side effect of introducting the "exports" section in package.json. 

It would probably be OK to add an export field for the workers. Maybe even skip the `dist`? 

`{
 "exports": {
    ".": {...},
    "./mvt-worker.js": {
      "import": "./dist/mvt-worker.js"
    }
  }
} 

Feel free to open a PR on the mvt module. We can do this module by module for now and do a pass on all modules once the idea is proven.
